pkg/edgedictionaryitem/describe.go

```diff
@@ -0,0 +1,49 @@
+package edgedictionaryitem
+
+import (
+	"io"
+
+	"github.com/fastly/cli/pkg/common"
+	"github.com/fastly/cli/pkg/compute/manifest"
+	"github.com/fastly/cli/pkg/config"
+	"github.com/fastly/cli/pkg/errors"
+	"github.com/fastly/cli/pkg/text"
+	"github.com/fastly/go-fastly/v2/fastly"
+)
+
+// DescribeCommand calls the Fastly API to describe a service.
+type DescribeCommand struct {
+	common.Base
+	manifest manifest.Data
+	Input    fastly.GetDictionaryItemInput
+}
+
+// NewDescribeCommand returns a usable command registered under the parent.
+func NewDescribeCommand(parent common.Registerer, globals *config.Data) *DescribeCommand {
+	var c DescribeCommand
+	c.Globals = globals
+	c.manifest.File.Read(manifest.Filename)
+	c.CmdClause = parent.Command("describe", "Show detailed information about a Fastly edge dictionary item").Alias("get")
+	c.CmdClause.Flag("service-id", "Service ID").Short('s').StringVar(&c.manifest.Flag.ServiceID)
+	c.CmdClause.Flag("name", "Name of dictionary").Short('n').Required().StringVar(&c.Input.DictionaryID)
```

Comment:
This should probably more closely align with Go-Fastly and the API in that it's actually an ID and not a name.

```suggestion
	c.CmdClause.Flag("dictionary-id", "Dictionary ID").Short('d').Required().StringVar(&c.Input.DictionaryID)
```

Reply:
BTW if I missed this 'name' vs 'id' problem any where else could you just presume a similar change is necessary there too.

**UPDATE**: I've just noticed in another file for listing dictionaries we do actually use `dictionary-id` as a flag.

Reply:
Yea...this was my mistake copy/pasting when some actions take a name and some take an ID.